control-base/src/main/java/fi/nls/oskari/control/view/modifier/bundle/MapfullHandler.java

```diff
@@ -109,7 +109,8 @@ public boolean modifyBundle(final ModifierParams params) throws ModifierExceptio
                 params.getViewType(),
                 bundleIds,
                 useDirectURLForMyplaces,
-                params.isModifyURLs());
+                params.isModifyURLs(),
+                mapSRS);
 
 
         // transform WKT for layers now that we know SRS
```

Comment:
Now that getFullLayerConfig() already adds the coverage information and transforms it the loop below will do it again and accumulate a massive amount of JSON:
![transforms_gone_wrong](https://user-images.githubusercontent.com/2210335/37213721-90e1b776-23bb-11e8-9e36-c3068742ab70.png)

Reply:
I'll fix this ^